apps/backend/scripts/db-migrations.ts

```diff
@@ -151,6 +152,8 @@ const migrate = async (selectedMigrationFiles?: { migrationName: string, sql: st
 
   console.log('='.repeat(60) + '\n');
 
+  await runClickhouseMigrations();
```

Comment:
maybe we can clarify in the console logs here and of the old migration that the old migration is the Postgres migration, and this is the Clickhouse migration?